policyengine_us/parameters/gov/states/az/tax/income/credits/non_refundable.yaml

```diff
@@ -3,6 +3,10 @@ values:
   2019-01-01:
     - az_dependent_tax_credit
     - az_family_tax_credit
+  2023-01-01:
+    - az_one_time_families_tax_rebates 
```

Comment:
Does the rebate apply in 2024 and on? if not then we need to remove it starting in 2024